app/models/user.rb

```diff
@@ -5,6 +5,7 @@ class User < ActiveRecord::Base
          :recoverable, :rememberable, :trackable, :validatable
 
   has_many :tattoos
+  has_many :reviews
```

Comment:
Same here as above - this should be reflected in the User unit test https://learn.launchacademy.com/questions/314